tests/tools/test_samtools.yml

```diff
@@ -0,0 +1,94 @@
+- name: quickcheck_miniwdl
+  tags:
+    - miniwdl
+    - samtools
+  command: >-
+    miniwdl run -d test-output/. --copy-input-files --task quickcheck tools/samtools.wdl bam="tests/tools/input/test.bwa_aln_pe.bam"
+  files:
+    - path: test-output/out/checked_bam/test.bwa_aln_pe.bam
+
+- name: split_miniwdl
+  tags:
+    - miniwdl
+    - samtools
+  command: >-
+    miniwdl run -d test-output/. --copy-input-files --task split tools/samtools.wdl bam="https://github.com/stjude/CICERO/raw/master/test/data/input/test.bam"
+  files:
+    - path: test-output/out/split_bams/0/
+    - path: test-output/out/split_bams/1/
```

Comment:
Are these supposed to be just directories and not full paths?

Reply:
Good catch, I changed the input file right before I opened the PR. Apparently, I left off the updated file names.